lib/tfctl/generator.rb

```diff
@@ -33,7 +34,7 @@ def make(account:, config:)
                     'backend'            => {
                         's3' => {
                             'bucket'         => config[:tf_state_bucket],
-                            'key'            => "#{account[:name]}/tfstate",
+                            'key'            => [tf_state_prefix, account[:name], 'tfstate'].join('/'),
```

Comment:
This creates an issue when `tf_state_prefix` isn't set.  It'll prefix the object key with a `/` which isn't valid:

```
error: test-example1: │ Error: key must not start with '/'
```
```
>> ['', 'account', 'tfstate'].join('/')
=> "/account/tfstate"
```

You could use `.delete_prefix('/')` to remove it:

```
>> ['', 'account', 'tfstate'].join('/').delete_prefix('/')
=> "account/tfstate"
>> ['prefix', 'account', 'tfstate'].join('/').delete_prefix('/')
=> "prefix/account/tfstate"
```

Reply:
Cool. Fixed.